release-notes/release-notes/5.8-release-notes.md

```diff
@@ -47,6 +52,14 @@ You can now pull specific data from REST API response headers using .NET regex c
 
 - We added a Listener section to the MongoDB Collection (Cinchy Event Triggered) and REST API (Cinchy Event Triggered) Sources. You can now manage the event trigger within the Connections UI. This reduces the complexity of managing the Listener Config table.
 
+### DataPollingConcurrencyIndex improvements
+
+We improved the implementation of `DataPollingConcurrencyIndex`. Additionally, we added logging around the **Acquire** and **Release** processes to enhance monitoring.
+
+### Model support for Cinchy Table Target
```

Comment:
CIN-06631. Need more information on this ticket. What, specifically, are the models. I can guess, but it's better to be explicit.